frontend/src/components/nutrigenomics/PathwayCard.tsx

```
@@ -0,0 +1,128 @@
+        config.border,
+        selected && "ring-2 ring-primary",
+      )}
+      onClick={onClick}
+      onKeyDown={(e) => {
+        if (e.key === "Enter" || e.key === " ") {
+          e.preventDefault()
+          onClick()
+        }
+      }}
+      tabIndex={0}
+      role="button"
```

Biome (2.4.6)

[error] The elements with this role can be changed to the following elements:
<button>

(lint/a11y/useSemanticElements)